dotCMS/src/test/java/v2/languages/LanguagesResourceTest.java

```diff
@@ -38,8 +41,74 @@ public void list(){
         when(languageAPI.getLanguages()).thenReturn(languages);
 
         final LanguagesResource languagesResource = new LanguagesResource(languageAPI, webResource);
-        final Response response = languagesResource.list(request);
+        final Response response = languagesResource.list(request, null);
 
         assertEquals(languages, ((ResponseEntityView) response.getEntity()).getEntity());
     }
+
+    @Test
+    public void test_when_contenInodeIsNotNull_should_listLanguageByInode() throws DotSecurityException, DotDataException {
+        final HttpServletRequest request  = mock(HttpServletRequest.class);
+        final WebResource webResource = mock(WebResource.class);
+        final LanguageAPI languageAPI = mock(LanguageAPI.class);
+        final InitDataObject initDataObject = mock(InitDataObject.class);
+        final User user = new User();
+        final List<Language> languages = CollectionsUtils.list(mock(Language.class));
+
+        when(initDataObject.getUser()).thenReturn(user);
+        when(webResource.init(null, true, request, true, null)).thenReturn(initDataObject);
+        when(languageAPI.getAvailableContentPageLanguages("2", user)).thenReturn(languages);
+
+        final LanguagesResource languagesResource = new LanguagesResource(languageAPI, webResource);
+        final Response response = languagesResource.list(request, "2");
+
+        assertEquals(languages, ((ResponseEntityView) response.getEntity()).getEntity());
+    }
+
+    @Test
+    public void test_when_contenInodeIsNotNull_and_throwDotDataException() throws DotSecurityException, DotDataException {
+        final HttpServletRequest request  = mock(HttpServletRequest.class);
+        final WebResource webResource = mock(WebResource.class);
+        final LanguageAPI languageAPI = mock(LanguageAPI.class);
+        final InitDataObject initDataObject = mock(InitDataObject.class);
+        final User user = new User();
+        final List<Language> languages = CollectionsUtils.list(mock(Language.class));
+        final DotDataException exception = mock(DotDataException.class);
+
+        when(initDataObject.getUser()).thenReturn(user);
+        when(webResource.init(null, true, request, true, null)).thenReturn(initDataObject);
+        when(languageAPI.getAvailableContentPageLanguages("2", user)).thenThrow(exception);
+
+        final LanguagesResource languagesResource = new LanguagesResource(languageAPI, webResource);
+
+        try {
+            languagesResource.list(request, "2");
+            assertTrue(false);
+        } catch(DotDataException e) {
+            assertTrue(true);
+        }
+    }
+
+    public void test_when_contenInodeIsNotNull_and_throwDotSecurityException() throws DotSecurityException, DotDataException {
```

Comment:
@Test is missing here. Also, you can take advantage of the expected param of @test when an exception is expected in your test, it's less verbose. Take a look at https://blog.goyello.com/2015/10/01/different-ways-of-testing-exceptions-in-java-and-junit/